code/game/gamemodes/gameticker.dm

```diff
@@ -465,6 +465,7 @@ var/global/datum/controller/gameticker/ticker
 
 /datum/controller/gameticker/proc/declare_completion()
 
+	nologevent = 1 //end of round murder and shenanigans are legal; there's no need to jam up attack logs past this point.

```

Comment:
There's only a single round end hook utilized....for mutiny---and setting it up would require more lines of code in a more non-obvious area than just placing it here. It also means that there will be a 5 second delay between the round ending and actual server attack logs being disabled for admins.